server/model/kino.go

```diff
@@ -0,0 +1,30 @@
+package model
+
+import (
+	"database/sql"
+	"time"
+)
+
+// Kino stores all movies
+type Kino struct {
+	Id          int32          `gorm:"primary_key;AUTO_INCREMENT;column:kino;type:int;not null;"`
```

Comment:
Please use int64 here - no need to restrict the id length here: https://gorm.io/docs/conventions.html